### PR TITLE
Update readme with release info

### PR DESCRIPTION
**Pull request by SteveLasker**
Signed-off-by: Steve Lasker <stevenlasker@hotmail.com>

### CODE REVIEW

**Review by shizhMSFT — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -4,7 +4,15 @@
...
-A [COSE](https://tools.ietf.org/html/rfc8152) library for go.
+A golang library for the [COSE specification][cose-spec]
+
+## Project Status
+
+**Current Release**: [go-cose alpha 1][release-alpha-1] 
+
+While the project is *currently* forked from the  upstream [mozilla/go-cose][mozilla-go-cose] project, the veraison and mozilla maintainers have agreed to decouple the fork, retire the mozilla/go-cose project and focus on [veraison/go-cose][veraions-go-cose] as the active project.
+
+We thank the [mozilla maintainers and contributors][mozilla-contributors] for their great work that formed the base of the [veraison/go-cose][veraions-go-cose] project.
```
Comment (yogeshbdeshpande) [inline README.md:15]:
```suggestion
We thank the [mozilla maintainers and contributors][mozilla-contributors] for their great work that formed the base of the [veraison/go-cose][veraion-go-cose] project.
```

File: README.md [hunk 2]
```diff
@@ -4,7 +4,15 @@
...
 [![coverage](https://github.com/veraison/go-cose/workflows/cover%20%E2%89%A589%25/badge.svg)](https://github.com/veraison/go-cose/actions?query=workflow%3A%22cover%20%E2%89%A589%25%22)
 
-A [COSE](https://tools.ietf.org/html/rfc8152) library for go.
+A golang library for the [COSE specification][cose-spec]
+
+## Project Status
+
+**Current Release**: [go-cose alpha 1][release-alpha-1] 
+
+While the project is *currently* forked from the  upstream [mozilla/go-cose][mozilla-go-cose] project, the veraison and mozilla maintainers have agreed to decouple the fork, retire the mozilla/go-cose project and focus on [veraison/go-cose][veraions-go-cose] as the active project.
```
Comment (yogeshbdeshpande) [inline README.md:13]:
```suggestion
While the project is *currently* forked from the  upstream [mozilla/go-cose][mozilla-go-cose] project, the veraison and mozilla maintainers have agreed to decouple the fork, retire the mozilla/go-cose project and focus on [veraison/go-cose][veraison-go-cose] as the active project.
```

File: README.md [hunk 3]
```diff
@@ -103,3 +111,9 @@ Fuzzing requires Go 1.18 or higher, and can be executed as follows:
 ```bash
 go test -fuzz=FuzzSign1
 ```
+
+[cose-spec]:            https://datatracker.ietf.org/doc/draft-ietf-cose-rfc8152bis-struct/
+[mozilla-contributors]: https://github.com/mozilla-services/go-cose/graphs/contributors
+[mozilla-go-cose]:      http://github.com/mozilla-services/go-cose
+[veraions-go-cose]:     https://github.com/veraison/go-cose
```
Comment (yogeshbdeshpande) [inline README.md:118]:
```suggestion
[veraison-go-cose]:     https://github.com/veraison/go-cose
```

**Review by yogeshbdeshpande — APPROVED** [review]

**Comment by thomas-fossati** [comment]
I am slightly confused.  In the README, we previously had warnings around the SignMessage interface being experimental.  Unless I'm looking at the wrong file, they seem to have disappeared?  Help me understand :-)

**Comment by yogeshbdeshpande** [comment]
> I am slightly confused. In the README, we previously had warnings around the SignMessage interface being experimental. Unless I'm looking at the wrong file, they seem to have disappeared? Help me understand 

@thomas-fossati  This has happened because Steve has raised a PR from this fork, which is not rebased to latest on master(main) [go-cose](https://github.com/SteveLasker/go-cose).

He need to do that before he merges to master and retain the change you have done in the previous commit.

**Comment by SteveLasker** [comment]
Thank you @yogeshbdeshpande

**Pull request merged**